utils/getClaimStatus.tsx

```diff
@@ -2,22 +2,65 @@
  * Utility file to get content for the Claim Status section.
  */
 
-import { ClaimStatusContent, I18nString } from '../types/common'
+import claimStatusJson from '../public/locales/en/claim-status.json'
+import { ClaimStatusContent, I18nString, TransLineProps } from '../types/common'
 import { ScenarioType } from './getScenarioContent'
+import getUrl, { UrlType } from './getUrl'
 
 /**
- * Get Claim Status description content.
+ * Helper function to get shared Claim Status translation string prefix.
  */
-export function getClaimStatusDescription(scenarioType: ScenarioType): I18nString {
-  return `claim-status:scenarios.${ScenarioType[scenarioType].toLowerCase()}.description`
+function getTranslationPrefix(scenarioType: ScenarioType): I18nString {
+  return `claim-status:scenarios.${ScenarioType[scenarioType].toLowerCase()}`
+}
+
+/**
+ * Get Claim Status heading.
+ */
+export function getClaimStatusHeading(scenarioType: ScenarioType): I18nString {
+  return getTranslationPrefix(scenarioType) + '.heading'
+}
+
+/**
+ * Get Claim Status summary.
+ *
+ * Note: This function has a lot of verbosity due to Typescript + eslint.
```

Comment:
Note: There's a lot going on in here to make the Typescript work properly.

Reply:
I feel like the casting isn't the right way to do this, digging in a bit

Reply:
nope, looks like this is right, but it does seem fragile. This feels like a probably anyone consuming an API would run into, I wonder what they've done?

Reply:
In the next PR, I've found a way to reduce the number of `as` casting to only cases where I'm trying to refer to using variables, so I think it's better.